Core/DISET/private/GatewayService.py

```diff
@@ -15,26 +15,35 @@
 
 """
 
-__RCSID__ = "$id:"
 
 import sys
 import cStringIO
+import os
+
+# TODO: Remove ThreadPool later 
+useThreadPoolExecutor = False
+if os.getenv('DIRAC_USE_NEWTHREADPOOL', 'NO').lower() in ('yes', 'true'):
```

Comment:
Please add an entry to docs/source/AdministratorGuide/ServerInstallations/environment_variable_configuration.rst